src/libs/ajax.js

```diff
@@ -944,6 +944,16 @@ const DataRepo = signal => ({
     //TODO: Update this link to hit the real endpoint
     const res = await fetchRawls(`dunno/what/this/is/${id}/requestAccess`, _.merge(authOpts(), { signal }))
     return res.json()
+  },
+
+  getPreviewMetadata: async id => {
+    const res = await fetchDataRepo(`repository/v1/snapshots/${id}?include=TABLES,DATA_PROJECT`, _.merge(authOpts(), { signal }))
+    return res.json()
+  },
```

Comment:
An alternative pattern you might consider to make these flow a bit better:
```suggestion
  getPreviewMetadata: id => {
    return fetchDataRepo(`repository/v1/snapshots/${id}?include=TABLES,DATA_PROJECT`, _.merge(authOpts(), { signal })).then(res => res.json())
  },
```

Reply:
I would prefer this to remain as-is, since the convention of the file is the way Christine coded it. If we want to change the convention, I would rather see a PR that goes through the whole file and changes it.

Reply:
It could be a bigger change in the future if we want to do that. I also just kind of think this reads better, but that's just me!